Wikipedia/Code/SideScrollingCollectionViewCell.swift

```diff
@@ -100,7 +104,8 @@ public class SideScrollingCollectionViewCell: CollectionViewCell, SubCellProtoco
         let widthToFit = size.width - layoutMargins.left - layoutMargins.right
         if !isImageViewHidden {
             if (apply) {
-                imageView.frame = CGRect(x: 0, y: 0, width: size.width, height: imageViewHeight)
+                let imageViewWidth = size.width - widthToFit > 50 ? widthToFit : size.width
```

Comment:
may be nice for future reference to use some sort of declarative naming for the frame bits here... something like `let saneAspectRatioFrame = `...